### ISSUE TITLE
spyder can not open

### ISSUE THREAD

**Issue by Peacefulyang** [spyder-ide/spyder#8779]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

    plugin = mod.PLUGIN_CLASS(self)
AttributeError: '_ModuleMock' object has no attribute 'PLUGIN_CLASS'

## Version and main components

* Spyder Version: 3.2.3 
* Python Version: 3.6.6
* Qt Versions: 5.9.6, PyQt5 5.9.2 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  2.0.0 (OK)
pycodestyle >=2.3:  2.4.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.23.4 (OK)
numpy >=1.7      :  1.15.1 (OK)
sphinx >=0.6.6   :  1.7.9 (OK)
rope >=0.9.4     :  0.11.0 (OK)
jedi >=0.9.0     :  0.12.1 (OK)
nbconvert >=4.0  :  5.4.0 (OK)
sympy >=0.7.3    :  1.2 (OK)
cython >=0.21    :  0.28.5 (OK)
qtconsole >=4.2.0:  4.4.1 (OK)
IPython >=4.0    :  6.5.0 (OK)
pylint >=0.25    :  2.1.1 (OK)

```

**Comment by danieltomasz** [comment]
Please update to the latest version of `Spyder`, this problem has been solved in one of the next releases (see https://github.com/spyder-ide/spyder/issues/7995)

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.3). Please update by opening the Anaconda Prompt and running there

    conda update spyder
